app/api/activitylog/activitylogMiddleware.js

```diff
@@ -39,6 +39,7 @@ export default (req, _res, next) => {
   if (mustBeLogged(baseurl, method, body)) {
     const time = Date.now();
     const expireAt = date.addYearsToCurrentDate(1);
+    if (body.password) body.password = '*****';
```

Comment:
I also removed this coz @daneryl noticed the password is displayed in plain text on the logs